pandas/_libs/tslib.pyx

```diff
@@ -92,6 +92,7 @@ from tslibs.nattype import NaT, nat_strings, iNaT
 from tslibs.nattype cimport _checknull_with_nat, NPY_NAT
 
 
+# TODO: Can this become a classmethod?
```

Comment:
@jreback @jorisvandenbossche

Reply:
no, see how its used, it is for compat on creating datetime, date, Timestamp